bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123021)
```suggestion
import React, { useEffect, useState } from "react";
```